phpcs-ruleset.xml

```diff
@@ -36,6 +37,7 @@
     <exclude-pattern>profiles/common/modules/custom/tmgmt_poetry/tests/</exclude-pattern>
     <exclude-pattern>build/vendor/</exclude-pattern>
     <exclude-pattern>profiles/multisite_drupal_standard/themes/atomium</exclude-pattern>
+    <exclude-pattern>profiles/multisite_drupal_standard/themes/europa</exclude-pattern>
     <exclude-pattern>profiles/multisite_drupal_communities/themes/atomium</exclude-pattern>
     <exclude-pattern>profiles/multisite_drupal_communities/themes/europa</exclude-pattern>
     <exclude-pattern>profiles/multisite_drupal_standard/themes/ec_resp/scripts</exclude-pattern>
```

Comment:
Concerning `ec_resp`, we exclude the entire repository for europa, should we not do the same for `ec_resp` as it is an external repository too; instead of pointing to specific folder like `scripts` or `css`? 
That would simplify the file.